cpo-docker/compose/cpo.yml

```diff
@@ -0,0 +1,41 @@
+---
+version: '2.4'
+
+services:
+  case-payment-orders-api:
+    image: "hmctspublic.azurecr.io/cpo/case-payment-orders-api:${CASE_PAYMENT_ORDERS_API_TAG:-latest}"
+    container_name: case-payment-orders-api
+    environment:
+      SERVER_PORT: 4457
+      IDAM_API_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      IDAM_OIDC_URL: "${IDAM_STUB_SERVICE_NAME:-http://idam-api:5000}"
+      SPRING_DATASOURCE_URL: jdbc:postgresql://cpo-db:5432/cpo
+      S2S_URL: http://service-auth-provider-api:8080
+      CPO_S2S_AUTHORISED_SERVICES: xui_webapp
+    ports:
+      - 4457:4457
+    depends_on:
+      - cpo-db
+    networks:
+      - ccd-network
+
+  cpo-db:
+    image: postgres:9.6
```

Comment:
Should be version 11 ?

Reply:
i have used one in ccd-docker. But yes, it's a good point.